src/common/styles/colors.tsx

```diff
@@ -7,6 +7,7 @@ export type ColorName =
   | "green"
   | "red";
 export type ToneLevel = 0 | 10 | 20 | 30 | 40 | 50 | 60 | 80 | 100;
+export type ToneLevelv2 = 0 | 10 | 20 | 30 | 40 | 50 | 60 | 70 | 80 | 90;
```

Comment:
casing wise should be `ToneLevelV2` (: Do update the case for the rest of the new variables and functions too.